src/geometry.jl

```diff
@@ -178,12 +178,14 @@ The first half level is at 0 the last at 1. Evaluate a generalised logistic func
 coefficients in `P` for the distribution of values in between. Default coefficients follow
 the L31 configuration historically used at ECMWF."""
 function vertical_coordinates(P::Parameters)
-    @unpack nlev,GLcoefs = P
+    @unpack nlev,GLcoefs,initialize_σ_manually,σ_levels_half = P
```

Comment:
then remove `initialize_σ_manually` here as that information is stored in the length of `σ_levels_half`